api-samples/tabCapture/background.js

```diff
@@ -0,0 +1,31 @@
+let currentReceiverTabId = null;
+
+chrome.action.onClicked.addListener(async (tab) => {
+  const currentTabId = tab.id;
+
+  if (currentReceiverTabId) {
+    await chrome.tabs.remove(currentReceiverTabId);
+  }
+
+  // Open a new tab with the receiver.html page
+  const { id: receiverTabId } = await chrome.tabs.create({
```

Comment:
Thoughts on using windows.create instead? This is sort of nice because you can have it open side by side and see the tab updating live. It also matches https://developer.chrome.com/docs/extensions/mv3/screen_capture/#audio-and-video.